src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -171,10 +240,14 @@ void RaftReplDev::on_push_data_received(intrusive< sisl::GenericRpcData >& rpc_d
     data_service()
```

Comment:
at line: 220, I don't quite understand while checking the DATA_RECEIVED, it will always be true and return, right?